### ISSUE TITLE
Feature: Power BI integration

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#3794]
As a ____, I want to _____ so that I can _____.

**In-scope:**

**Out-of-scope:**

**Resources:**
- Wireframe

**Data Model:**
_Brackets = relational data source
Parentheses = additional context/questions
Asterisks = required field for entry_

**Additional notes:**

**Comment by amenity** [comment]
Per conversation with @johnclary 8/27:
- Doesn’t make sense to invest too much in an integration until we have a real PowerBI user base
- Something like a “Open in PowerBI” button next to “Export as CSV” button might encourage users to take their analysis elsewhere. And maybe [fairly straightforward to implement](https://powerbi.microsoft.com/en-us/blog/using-a-power-bi-app-to-upload-csv-files-to-a-dataset/)?
- Could set up a dashboard or two that are built specifically for common Moped reporting needs

**Comment by johnclary** [comment]
Yep. And we might just rename this issue to "Socrata integration." And the data can be pulled into PowerBI from Socrata. I don't have experience using PowerBI + Scorata, but I believe @SurbhiBakshi has found it pretty straightforward. And Mooney built his 311 PowerBI dashboard from a Scocrata dataset.

**Comment by SurbhiBakshi** [comment]
@johnclary - it's fairly straightforward using the OData connector via PowerBI.

**Comment by amenity** [comment]
@SurbhiBakshi @johnclary — thanks! Renaming this to "Socrata integration" cc/ @zappfyllc @mateoclarke .

**Comment by amenity** [comment]
Closing my ol' Moped issues in the icebox. 👋 🛵 🥲
